content/en/platform/corda/4.4/enterprise/release-notes-enterprise.md

```diff
@@ -17,6 +17,37 @@ weight: 10
 
 # Corda Enterprise Release notes
 
+## Corda Enterprise 4.4.12
+
+Corda Enterprise 4.4.12 is a patch release of Corda Enterprise focused on security improvements.
+
+### Upgrade recommendation
+
+As a developer, you should upgrade to the [latest released version of Corda](../../../../../en/platform/corda/4.9/enterprise.html) as soon as possible. The latest Corda Enterprise release notes are on this page,
```

Comment:
Sorry, I should have been clearer.
Can we not say.. "As a developer or a node operator, you should upgrade.... " and remove the last sentence? Is the link to the upgrade guide not relevant to operators also?